ts/Dashboards/Components/Sync/Sync.ts

```diff
@@ -58,33 +65,69 @@ namespace Sync {
  * */
 class Sync {
```

Comment:
Question- Do we need to publically document that class?
Shouldn't it be marked as `internal`?